var/spack/repos/builtin/packages/gemmlowp/package.py

```diff
@@ -0,0 +1,40 @@
+#############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Gemmlowp(Package):
+    """Google low-precision matrix multiplication library"""
+
+    homepage = "https://github.com/google/gemmlowp"
+    version('a6f29d9ac', git='https://github.com/google/gemmlowp.git',
```

Comment:
The problem with calling it develop is, what happens when someone adds a newer commit? You can't also call that develop.

I think we should ask or encourage the developers to make releases. But obviously we have no control over that. For packages that don't make releases and expect all users to use the latest version, they often don't support problems in older commits. We should honor there wishes and simply install the latest develop or master branch, whichever they recommend.

I would also like to hear @citibeth's thoughts. I know there are security concerns about this approach.